pkg/policy/report.go

```diff
@@ -91,9 +91,6 @@ func generateFailEventsPerEr(log logr.Logger, er *response.EngineResponse) []eve
 	for i, rule := range er.PolicyResponse.Rules {
 		if rule.Status == response.RuleStatusPass {
 			continue
-		} else if rule.Status == response.RuleStatusSkip {
```

Comment:
Shouldn't we keep the if statement here ? If we just remove it we will fall in the else block and it will create events right ?